foundations/foundation-config/src/main/java/io/servicecomb/config/ConfigUtil.java

```diff
@@ -169,4 +201,45 @@ public static void installDynamicConfig() {
         AbstractConfiguration dynamicConfig = ConfigUtil.createDynamicConfig();
         ConfigurationManager.install(dynamicConfig);
     }
+
+  private static class ServiceCombPropertyUpdateListener implements WatchedUpdateListener {
+
+    private final ConcurrentMapConfiguration injectConfig;
+
+    ServiceCombPropertyUpdateListener(ConcurrentMapConfiguration injectConfig) {
+      this.injectConfig = injectConfig;
+    }
+
+    @Override
+    public void updateConfiguration(WatchedUpdateResult watchedUpdateResult) {
```

Comment:
will change all test case config property key to "servicecomb.cse.servicecomb.xxx"